src/components/Nav/Masthead/UserDropdown.tsx

```diff
@@ -122,6 +127,11 @@ class UserDropdownConnected extends React.Component<UserProps, UserState> {
           toggle={<DropdownToggle onToggle={this.onDropdownToggle}>{this.props.session.username}</DropdownToggle>}
           dropdownItems={[userDropdownItems]}
         />
+        {authenticationConfig.strategy === AuthStrategy.openshift && authenticationConfig.logoutEndpoint && (
+          <form id="openshiftlogout" action={authenticationConfig.logoutEndpoint} method="post">
+            <input type="hidden" name="then" value={authenticationConfig.logoutRedirect} />
+          </form>
+        )}
       </>
```

Comment:
Can we use a PF4 component here? If not please can you add a JIRA to the PF4 story and I'll Migrate this in another PR.